src/types.ts

```diff
@@ -52,184 +43,28 @@ export type ConfigSeedArgs = PartialConfigSeedArgs & {
   uuid: string,
 }
 
-export type AnyConfigBuilder = ConfigSeed | EitherInstancesConfig
-export type PlayerConfigs = ObjectS<ConfigSeed> | Array<ConfigSeed>
-export type MachineConfigs = ObjectS<PlayerConfigs>
-
-export type ScenarioConfig = {
-  players: PlayerConfigs,
-  initialization?: Initialization,
+// export type PlayerConfigs = ObjectS<PlayerConfig>
+export type PlayerConfig = ConfigSeed
+export type InstallHapps = AgentHapp[]
+export type AgentHapp = DnaPath | DnaPath[]
```

Comment:
This name doesn't make sense to me, what's an AgentHapp?

Reply:
It's an "App" ... but "App" is such a not useful name... 
I'm still very not clear on whether an 'App' is a 'Happ' or a 'HappBundle' or ...

Reply:
I started with `Agent...` because it is defined by the fact that these Dnas will become associated with an AgentPubKey (and an "app_id") to form an "App"